c/README.md

```diff
@@ -18,10 +18,11 @@ sudo apt-get install build-essential pkg-config cmake
 On other systems, you can obtain CMake from your operating system
 package manager or from http://www.cmake.org/.
 
-Once you have the dependencies installed, From the libsbp root
-directory, create a build directory where the library will be built:
+Once you have the dependencies installed,
```

Comment:
Hey @PaulBouchier: unfortunately these instructions are invalid if you're reading the README in the c directory. What about changing the existing text to `From this (the libsbp c) directory` or something.